tests/Node.py

```diff
@@ -6,6 +6,7 @@
 import re
 import json
 import signal
+import requests
```

Comment:
It looks like this isn't available on most of the buildkite pipelines.

Reply:
I'll fix PR after 2 PM EST. We need `pip install requests` for this. Do you know right place for this?

Reply:
I'm guessing in the .cicd pipeline files, but not sure where

Reply:
@brianjohnson5972 I'm already resolved this for Ubuntu. working for other platforms

Reply:
@brianjohnson5972 I'm having issues with `libpqxx` on `macos` . it seems like when rebuilding a base image it fails to install this library from particular commit. I tried to update the commit.
```
curl -LO  https://raw.githubusercontent.com/Homebrew/homebrew-core/106b4b8a421dda33c79a4018c3c3816234076331/Formula/libpqxx.rb
brew install -f ./libpqxx.rb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1442  100  1442    0     0   2089      0 --:--:-- --:--:-- --:--:--  2089
Warning: Calling `cellar` in a bottle block is deprecated! Use `brew style --fix` on the formula to update the style or use `sha256` with a `cellar:` argument instead.
Warning: Calling `sha256 "digest" => :tag` in a bottle block is deprecated! Use `brew style --fix` on the formula to update the style or use `sha256 tag: "digest"` instead.
Warning: Calling `sha256 "digest" => :tag` in a bottle block is deprecated! Use `brew style --fix` on the formula to update the style or use `sha256 tag: "digest"` instead.
Warning: Calling `sha256 "digest" => :tag` in a bottle block is deprecated! Use `brew style --fix` on the formula to update the style or use `sha256 tag: "digest"` instead.
Warning: Calling `sha256 "digest" => :tag` in a bottle block is deprecated! Use `brew style --fix` on the formula to update the style or use `sha256 tag: "digest"` instead.
Error: Failed to load cask: ./libpqxx.rb
Cask 'libpqxx' is unreadable: wrong constant name #<Class:0x00007fab95a8b540>
Warning: Treating ./libpqxx.rb as a formula.
==> Downloading https://ghcr.io/v2/homebrew/core/libpqxx/manifests/7.2.1
##O#- #
curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "libpqxx_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/libpqxx/manifests/7.2.1
```